src/main/java/com/cgdp/library/controller/AuthorController.java

```diff
@@ -0,0 +1,25 @@
+package com.cgdp.library.controller;
+
+import com.cgdp.library.entity.AuthorEntity;
+import com.cgdp.library.service.AuthorService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@AllArgsConstructor
+@RequestMapping("/authors")
+@RestController
+public class AuthorController {
+
+    private AuthorService authorService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public AuthorEntity newAuthor(@RequestBody AuthorEntity author) {
```

Comment:
We should not receive an entity as an input
We should have a explicit request object with the necessary data do do this operation, in this case is only the name of the author